### ISSUE TITLE
[469c] Game won't start after installing patch

### ISSUE THREAD

**Issue by rmysliwczyk** [OldUnreal/UnrealTournamentPatches#1173]
I've installed the patch on my Polish version of Unreal Tournament. I'm suspecting that it has something to do with the language version of my game, because the game doesn't want to start and shows a message box with error that says something about fonts:

Critical Error
Nie udało się wpisać Entry: Nie odnaleziono Function w 'Function Engine.Canvas.CreateFont'
Historia: UGameEngine::Init <-InitEngine

this would translate roughly to:

Critical Error
Failed to enter Entry: Function not found in 'Function Engine.Canvas.CreateFont'
History: UGameEngine::Init <-InitEngine

**Comment by an-eternity** [comment]
Probably same problem here: https://github.com/OldUnreal/UnrealTournamentPatches/issues/1143

**Comment by rmysliwczyk** [comment]
> Probably same problem here: #1143

Thank you, that's correct. I have the same version. Unfortunately it seems I won't be able to use my version. I really wanted to play on some private servers, but when I connect to any server they throw error about the "Engine" version not matching. I thought that updating to this patch would solve the issue. I now begin to think that it's not about incorrect version (I'm on 436), but rather there were some more changes done to the game files during localization process, and that's why I can't connect, no idea. It was worth a shot though.

**Comment by rmysliwczyk** [comment]
I've found a solution to make this copy/version work! I will post it in the issue #1143 too. I managed to get the patch working by deleting all of the files that end with .pol and .pol_u from System folder in UT installation. This will remove the Polish localization, but all of the original files are still present.  There is probably some more elegant solution that involves changing some config file to look for the original files instead of those, but it works!
